includes/oauthhelper/class-oauth-service.php

```
@@ -73,30 +73,15 @@ public function add_auth_via_xtendify_button()
 		$query_arg = [
 			'request_from' => $site_url,
-			'state' => wp_create_nonce('simcal_oauth_via_sc_state'),
 		];
 		$authredirect = add_query_arg($query_arg, SIMPLE_CALENDAR_OAUTH_HELPER_AUTH_DOMAIN . 'helper/');
 		$admin_image_about_path = SIMPLE_CALENDAR_ASSETS . '/images';
 
 		if (!empty($_GET['auth_token'])) {
 			$auth_token = sanitize_text_field((string) wp_unslash($_GET['auth_token']));
-			$state = isset($_GET['state']) ? sanitize_text_field((string) wp_unslash($_GET['state'])) : '';
-
 			$helper_origin_ok = false;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$helper_origin_ok'. (undefined)

(UnusedLocalVariable)